runtime/compiler/runtime/J9Profiler.hpp

```diff
@@ -174,8 +177,16 @@ class TR_PersistentProfileInfo
 
    void dumpInfo(TR::FILE *);
 
-   private:
+   void setCallSiteInfo(TR::Compilation *comp, uint32_t numCallSites, TR_InlinedCallSite *inlinedCallSite);
+   void getSerializedSize(TR_Serializer &serializer) const;
```

Comment:
Can you please add doxygen for newly added APIs to explain what they do, their requirements, results and side-effects?